app/models/Document.js

```diff
@@ -24,7 +26,7 @@ export default class Document extends BaseModel {
 
   collaborators: User[];
   collectionId: string;
-  lastViewedAt: ?string;
+  @observable lastViewedAt: ?string;
```

Comment:
```suggestion
  lastViewedAt: ?string;
```

`observable` shouldn't be needed here, anything that comes down in the JSON payload is automatically observable

Reply:
I had to add it.. because it wasn't working without it

Reply:
Also, i need the observable because i update this property when a user enters the document, without passing by the api..

Reply:
Weird, it should be automatically observable – I must be misunderstanding something

Reply:
When we dispatch the action to set a document as viewed.. I added a method `document.updateLastViewed(view)` that updates the object stored on memory, setting the lastView property.. Without the @observable, the update wasn't reflecting on the screen.